### DISCUSSION TITLE
Reinstalled on new SSD.  Can't find mapping profile.

### DISCUSSION THREAD

**Discussion by mcold1021** [rsjaffe/MIDI2LR#738] [Q&A]
Hi...I replaced my SSD and am stumped about where MIDI2LR saves the key mapping profile.  MIDI2LR opens with Lightroom and displays "Connected to Lightroom" but is completely blank under MIDI Command & LR Command headers.  No key mapping. 

What should I be looking for?

Thanks!

**Comment by antzpantz** [comment]
If I remember correctly, the default location for saving your key bindings
(.xml file) was My Documents which is in C:\Users\<username>\Documents

**Reply by mcold1021** [reply]
I scanned the Documents folder from the backup of my legacy SSD, for *.xml files. Nothing. Does MIDI2LR assign the name of the key bindings file or does the user?  If the former, do you know the name?  So frustrating. I backed up everything. The file has to be there.  Thanks for you quick response to my post.

**Comment by GregEakin** [comment]
Mine are stored in the `C:\Users\greg\OneDrive\Documents\MIDI2LR\` folder.

**Comment by donluca** [comment]
Same problem here, but on macOS.

Does anyone know where is the default profile save folder on macOS?

**Comment by antzpantz** [comment]
I found this in an old thread here, hopefully it helps:

"In Windows, you will find the file in the %appdata%\MIDI2LR directory
(type that in the address bar of the file explorer to jump to the correct
directory). On the Mac, it will be at ~/Library/Application
Support/MIDI2LR/. Also, edit your profiles containing the old commands
prior to starting MIDI2LR."

https://groups.google.com/g/midi2lr/c/rXY8ktJjfvY/m/v6ZMy71LDAAJ

**Reply by donluca** [reply]
Thank you so much!

I'll look into this later and report back!

**Reply by donluca** [reply]
It worked!
It looked like I had used a custom folder for my profiles, maganed to find it and restored my profile, thank god.
I had so many mappings in there!

Thanks again!

**Comment by antzpantz** [comment]
Glad to hear and glad to help :)
